index.js

```diff
@@ -106,7 +106,7 @@ function generateSignature(
     if (securityPacket.user_id) {
         signatureArray.push(securityPacket.user_id);
     }
-    signatureArray.push(secret);
+    // signatureArray.push(secret);
```

Comment:
@bhavya-shukla-lrn This can be removed fully.

Reply:
Removed Comment